MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -4356,10 +4322,39 @@
     {
       "Id": "ChestWoodfallRedRupee",
       "RequiredItems": [
-        "AreaSouthAccess",
-        "Woodfall Platforming"
+        "AreaSouthAccess"
+      ],
+      "ConditionalItems": [
+        [
+          "MaskDeku"
+        ],
+        [
+          "FD Difficult Jumps"
```

Comment:
Should this and the above be "Woodfall Platforming"? Or did you intend to remove that grouping?

Reply:
See above